google-cloud-clients/google-cloud-bigtable-admin/README.md

```diff
@@ -0,0 +1,134 @@
+# Google Cloud Java Client for Bigtable Admin
```

Comment:
Add a link to google-cloud-bigtable-admin from the root README

Reply:
done